### PR TITLE
Ghost TX Lite

### PR DESCRIPTION
**Pull request by brandonrc**
- [x] Power levels, nothing changes. Maybe try and hardcode the value? 
- [ ] Packet rate display only showing `0:100`

### CODE REVIEW

**Comment by JyeSmith** [comment]
>     * [ ]  Packet rate display only showing `0:100`

I also get this and it appears to be an issue with the ExLRS bin.  The official nightly only displays 0:250, which is expected due to syncing not being in the nightly. 

https://github.com/ExpressLRS/ExpressLRS/pull/286#issuecomment-803513189

File: src/platformio.ini [hunk 1]
```diff
@@ -391,6 +391,24 @@ upload_flags =
...
 	VECT_OFFSET=0x4000
 
+[env:GHOST_2400_TX_LITE_via_STLINK]
+extends = env_common_stm32
+platform = ststm32@11.0.0
+board = GHOST_TX
+build_flags =
+	${env_common_stm32.build_flags}
+	${common_env_data.build_flags_tx}
+ 	-D TARGET_TX_GHOST_LITE
```
Comment (JyeSmith) [inline src/platformio.ini:401]: I wonder if you can also include -D TARGET_TX_GHOST here, then have an elif for the different power settings.  This should remove the need for all of the other TARGET_TX_GHOST_LITE additions around the code.

**Review by JyeSmith — COMMENTED** [review]
Nice work!

File: src/platformio.ini (hunk 1 above)
Comment (brandonrc) [reply src/platformio.ini:401]: Good idea! Cleans it up a bit. I did not know if I needed to tweak anything else. 

RIght now I cannot get the power to play nice... something is off. Maybe we can talk about it on discord but I might have to do some probing.

**Pull request merged**